downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -70,6 +96,49 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 #### OpenAPI tool
 
+#### Persist tool
+- Modified the `persist init` command to solely create a `persist` directory within the Ballerina project and generate a new definition file (`model.bal`) within the `persist` directory if it doesn't already exist. It no longer updates the `Ballerina.toml` file with the persist configuration as it did previously.
+- Modified the `persist generate` command to function as a one-time source code generation tool. Additionally, introduced the following new arguments to the `persist generate` command:
```

Comment:
These are CLI options, right? I think we should use "options" instead of "arguments". Applicable in other places too.

Reply:
Fixed in [70ca3c7](https://github.com/ballerina-platform/ballerina-dev-website/pull/8881/commits/70ca3c7b388d2526aa364810f78adc5a3f47dd67)